tritium-processor/src/main/java/com/palantir/tritium/processor/TritiumAnnotationProcessor.java

```diff
@@ -150,66 +128,17 @@ public boolean process(Set<? extends TypeElement> _annotations, RoundEnvironment
         return false;
     }
 
-    private Set<Element> getElementsToProcess(RoundEnvironment env) {
-        Set<Element> currentElements = new HashSet<>(env.getElementsAnnotatedWith(Instrument.class));
-        for (Name name : invalidElements) {
-            currentElements.add(elements.getTypeElement(name));
-        }
-        invalidElements.clear();
-        return currentElements;
-    }
-
-    private boolean isInvalid(
-            List<? extends TypeMirror> mirrors,
-            List<DeclaredType> allInterfaces,
-            @Nullable List<DeclaredType> minimalInterfaces) {
-        for (TypeMirror mirror : mirrors) {
-            if (isInvalid(mirror, allInterfaces, minimalInterfaces)) {
-                return true;
-            }
-        }
-        return false;
-    }
-
-    /**
-     * Returns true if the given {@link TypeMirror} has sufficient type information, otherwise it may need to be
-     * deferred for another annotation processor to complete.
-     */
```

Comment:
Is there a particular reason why we need to handle this case here but not in our other delegate-generating annotation processors that use `MoreElements`?